File: Patterns/02. Rectangle Patterns/15. Rectangle Odd Numbers/ron.cpp
#include <iostream>
using namespace std;
int main(){
    int l,b;
    cout<<"Enter the Length : ";
    cin>>l;
    cout<<"Enter the Breadth : ";
    cin>>b;

    for(int i=1; i<=l; i++){
        for(int j=1; j<=b; j++){
            cout<<2*j-1<<" ";
        }
        cout<<'\n';
    }
}

/*
Output

1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13  

*/

File: Patterns/02. Rectangle Patterns/15. Rectangle Odd Numbers/ron.java
import java.util.*;
public class ron{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the Length : ");
        int l = sc.nextInt();

        System.out.print("Enter the Breadth : ");
        int b = sc.nextInt();

        for(int i=1; i<=l; i++){
            for(int j=1; j<=b; j++){
                System.out.print(2*j-1 + " ");
            }
            System.out.println();
        }
        sc.close();
    }
}

/*
Output

1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13 
1 3 5 7 9 11 13   

*/